AppController/terminate.rb

```diff
@@ -7,18 +7,19 @@
 APPSCALE_CONFIG_DIR = "/etc/appscale"
 
 module TerminateHelper
-
-
   # Erases all AppScale-related files (except database state) from the local
   # filesystem. This is used when appscale is shutdown.
   # TODO: Use FileUtils.rm_rf instead of backticks throughout this
   # method.
   def self.erase_appscale_state
+    `service appscale-controller stop`
```

Comment:
monit will probably just restart the controller right after this. I don't think it's needed since `appscale-stop-services` should stop the controller.